app/src/main/res/layout/language_item_new.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I believe this root element is clickable. We should annotate it as such through `android:clickable` and `android:focusable`